f8a_worker/models.py

```diff
@@ -388,3 +388,14 @@ class DownstreamMap(Base):
 
     key = Column(String(255), primary_key=True)
     value = Column(String(512))
+
+
+class RecommendationFeedback(Base):
+    __tablename__ = "recommendation_feedback"
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    stack_id = Column(String(64), nullable=False)
+    package_name = Column(String(255), nullable=False)
+    recommendation_type = Column(String(255), nullable=False)
+    feedback_type = Column(Boolean, nullable=False, default=False)
+    ecosystem = Column(String(64), nullable=False)
```

Comment:
Shouldn't `ecosystem` be a foreign key pointing to `StackAnalysisRequest` table (?) And the same for `package_name`?

Reply:
@msrb this is where I need help. Ecosystem will be a foreign key and the Stackid will also be a ForeignKey. 
Package Name should not be because we will always have a delta between the data synced at the graph and the one for PGM training. So we can let package name, but we need relations for ecosystem and Stackid. I could not figure how to add those.